wire-ios-sync-engine/Tests/Source/Synchronization/ZMHotFixTests.swift

```diff
@@ -246,44 +246,44 @@ final class ZMHotFixTests_Integration: MessagingTest {
     }
 
     func testThatItUpdatesAccessRolesForConversations_432_1_0() {
```

Comment:
This flaky test is (hopefully) fixed now: the flaky cause seemed to add the notification observer after performing already the logic. 
Fix: the observer is added first.